### PR TITLE
Add previous/next navigation buttons to the filter bar.

### PR DESCRIPTION
**Pull request by troglodite2**
Currently dockets have upto 200 entries on them.  We have navigation buttons at the very bottom that allow navigation.  Many people use scrolling to move through a page and a 200 item docket can be time consuming.

We add Previous and Next buttons to the filter bar to allow people to navigate more easily when at the top of the page.

During testing it became apparent that the Bootstrap XS version was pretty ugly, per CL design specifications.  Code was added to override.css to make things look better.

When testing at a screen width of 360px it again looked horrible. Having things look right at XS width and at 360 wasn't possible.

To allow for another breakpoint a series of classes were added to override.css that provide cl-col-xxs- and cl-col-xs- as well as some auxilary classes.  These allow for that extra breakpoint.

the cl-col-xs- are required because the col-xs- classes are always "on" and we needed a method to override them.

We also added some specific cl-pull classes to allow different alignment based on screen size.

Fixes: #2670

### CODE REVIEW

**Comment by mlissner** [comment]
What happened to the boostrap-sm that you had here: https://github.com/freelawproject/courtlistener/issues/2670#issuecomment-1524120128

**Comment by troglodite2** [comment]
I can try to get back to it.  I think going from small to large, I kept the wrong look.

**Comment by troglodite2** [comment]
Per discussions, we are designing this for standard bootstrap break points.  The 360px is what it is.
360px:
image
Bootstrap xs: 767px:
image
Bootstrap sm: 991px
image
Bootstrap md: 1199px
image
Bootstrap lg and desktop: 1500px
image

Once these images are approved, I'll push up the next commit.

**Comment by mlissner** [comment]
These look good, but the screenshots aren't the most difficult widths. 767, 991, and 1199 are the best cases for xs, sm, and md sizes. The harder question is one pixel bigger: The smallest size before the breakpoint kicks in. Can you share 768, 992, and 1200? 

My general feedback though is we should try to stick to two lines max, except on xs, and should drop pagination on the smaller sizes if we need to, to make that happen.

**Comment by troglodite2** [comment]
768px:
image
992px:
image
1200px:
image

**Comment by mlissner** [comment]
Cool. Those look great. Do what you can for xs, and let's ship.

File: cl/assets/static-global/css/override.css [hunk 1]
```diff
@@ -1428,3 +1436,21 @@ textarea {
...
+  margin-bottom: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+@media (max-width: 768px) {
+  .cl-xs-pull-left {
+    float: left !important;
+  }
+  .cl-xs-pull-right {
+    float: right !important;
```
Comment (mlissner) [inline cl/assets/static-global/css/override.css:1454]: I don't think we need to namespace these, and it seems like the convention in BS4 and BS5 is to call them `pull-right-xs` and and `pull-left-xs`.

**Review by mlissner — CHANGES_REQUESTED** [review]
A couple little CSS things, but we're on the homestretch.

**Review by mlissner — APPROVED** [review]

**Pull request merged**